blog/docs/articles/getting-started-blog-p2.md

```diff
@@ -0,0 +1,266 @@
+(which actually has all the language support that VSCode has if you are willing to configure it yourself). While your choise of editor doesn't impact the quality of code
+which you will produce, it is crucial that you choose one and become comfortable working in it as much of your productivity will come from your ability to use your editor.
+
+### Running Knative Locally
+
+Now that you have verified that you have all the prerequisites, let's get started setting up your local system to run Knative!
+
+#### Docker/Podman Setup
+
+Knative runs in a Kubernetes Cluster, which means it runs as a set of Containers. You can think of a container as something which both contains your code and all the
+dependencies needed to run your code, as well as something which runs your code with those packaged dependencies. The two most common tools used to build and run containers
+are __Docker__ and __Podman__, so we will discuss setting both up here. However, do note that we think that __using Docker__ is easier due to its direct integration with a
+container registry. As such, we recommend that you use Docker.
+
+##### Option 1: Setting up Docker
+
+The first step to setting up Docker is to install it. The easiest way to do this is to [install Docker Desktop](https://docs.docker.com/get-docker/), although you only need
+the Docker Engine for Knative so you can also just [install the Docker Engine](https://docs.docker.com/engine/install/). Note: if you plan on just installing the Docker Engine,
+make sure that you complete the post-install steps.
+
+Once you have Docker installed on your system, you will need to log into a docker account in order to connect to a docker registry later (this is a common step when building
+Knative). If you need to create an account, you can [sign up for one here](https://hub.docker.com/signup). Once you have an account, all you need to do is type 
+`docker login`, and follow the instructions.
+
+##### Option 2: Setting up Podman
+
+If you want to use Podman instead of Docker, you're in luck - they have compatible CLIs! So, all you have to do is [install Podman](https://podman.io/docs/installation)
+and then alias docker to podman. To create this alias you must:
+
+- If you are using bash, add `alias docker=podman` to your `.bashrc` file.
+
+- If you are using zsh, add `alias docker=podman` to your `.zshrc` file.
+
+- If you are using fish, add `alias docker=podman` to your `config.fish` file.
+
+Similar to setting up Docker, once you have Podman installed onto your system aand your alias set up, you will need to have a docker registry account and run `docker login`.
+However, you will need to provide which registry to log in to. So, you will need to run `docker login docker.io`. Once again, note that we recommend using Docker instead of
+Podman.
+
+#### Minikube Setup
```

Comment:
In a lot of scripts (e.g. https://github.com/knative/eventing/blob/main/hack/create-kind-cluster.sh) kind is used instead of minikube.
I think it would be helpful to add some information about Kind here too.